core/kazoo_services/src/services/kz_service_phone_numbers.erl

```diff
@@ -57,6 +58,18 @@ reconcile(Services, PNs) ->
     S2 = kz_services:reset_category(?NUMBER_SERVICES, S1),
     update_numbers(S2, PNs).
 
+-spec reset(kz_services:services()) -> kz_services:services().
+reset(Services) ->
+    Categories = [?PHONE_NUMBERS
+                 ,?NUMBER_SERVICES
```

Comment:
@fenollp these were taken from reconcile/2 but feel free to update the PR.
note that, currently this is crashing when an account is created as reconcile gets called but there are no numbers to reconcile.